monitoring/oplog-populator/alerts.yaml

```diff
@@ -36,6 +39,18 @@ groups:
       description: "Oplog populator failed to configure connector"
       summary: "Oplog populator couldn't update kafka connect connector"
 
+  - alert: KafkaConnectTooManyConnectors
+    Expr: |
+        sum(s3_oplog_populator_connectors{job="${oplog_populator_job}",namespace="${namespace}"})
+        > ${oplogPopulatorChangeStreamLimit}
+    For: "5m"
+    Labels:
+     severity: warning
+    Annotations:
+      zenko_service: backbeat-oplog-populator
+      description: "Kafka connector count is above the soft limit"
+      summary: "Kafka connector count is above the soft limit. If the number of buckets is too high, consider using a single replica for your cluster"
+
```

Comment:
after an upgrade, we may have buckets we unexpectedly listen to (not just as "temporary" retained), but longer term : e.g. we remove a workflow rule, but the connector is used for multiple buckets

maybe we want to alert when the number of retained buckets is stable for "some time" (a few minutes).

this also means we may not want to update the metric continuously, but possibly only at the end of the "reconciliation" : having retained bucket in the middle of the reconciliation is expected and normal (though unlikely), having them after the reconciliation means we listen to more buckets than we should...

(or maybe we can just ignore it, since this is kind of normal and not really an issue? the only "risk" i see if a user tries to disable workflow on a high-traffic bucket to lower the load via oplog, and this kind of situation prevents the change to be effective....)